docker/README.md

```diff
@@ -124,6 +146,12 @@ Then build it with:
 $ docker build -t company/appname:tag .
 ```
 
+And run:
+
+```console
+$ docker run --name appname-i-001 -p 3301:3301 -d company/appname:tag
```

Comment:
1. We use `mytarantool` everywhere for the container name. Could you please change it to `mytarantool-001`?
2. Looks like `-e TT_INSTANCE_NAME=instance-001` is missing.

Reply:
1. Changed to `app-instance-001`.
2. Sure. Thanks for the catch!